types/parse/index.d.ts

```diff
@@ -913,18 +913,18 @@ namespace Parse {
          */
         addIndex(name: string, index: Schema.Index): this;
```

Comment:
`Schema.Index` will make type error cause 983 line.

Reply:
I didn't get this one... what should I write here?

Reply:
It will be fixed naturally when you fix the 983 line as `type TYPE = 'String' | 'Number' | 'Boolean' | 'Date' | 'File' | 'GeoPoint' | 'Polygon' | 'Array' | 'Object' | 'Pointer' | 'Relation'`